letsencrypt/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+letsencrypt_webroot: "/var/www/letsencrypt"
+letsencrypt_webuser: "www-data"
+letsencrypt_webserver: "nginx"
+letsencrypt_webserver_config: "nginx/letsencrypt"
+letsencrypt_webserver_sites_available: "/etc/nginx/sites-available"
+letsencrypt_webserver_sites_enabled: "/etc/nginx/sites-enabled"
+letsencrypt_https_redirect: false
+letsencrypt_email: !!null
```

Comment:
Yep, I set this in our Ficus base vars: https://github.com/noderabbit-team/edx-configs/pull/75/files
With this being a public repo, I'd prefer not to get notices about other peoples' expired certs :p